src/js/skipto.js

```diff
@@ -186,26 +201,60 @@
       this.buttonNode.setAttribute('accesskey', this.config.accesskey);
 
-      if (this.isNotEmptyString(this.config.buttonTitleWithAccesskey) &&
-        (this.config.accesskey.length === 1)) {
-        var title = this.config.buttonTitleWithAccesskey.replace('$key', this.getBrowserSpecificAccesskey(this.config.accesskey));
-        this.buttonNode.setAttribute('title', title);
-      } else {
-        if (this.isNotEmptyString(this.config.buttonTitle)) {
-          this.buttonNode.setAttribute('title', this.config.buttonTitle);
-        }
-      }
-
       this.domNode.appendChild(this.buttonNode);
+
+      this.renderTooltip(this.domNode, this.buttonNode);
+
       this.menuNode = document.createElement('div');
       this.menuNode.setAttribute('role', 'menu');
       this.domNode.appendChild(this.menuNode);
       this.buttonNode.addEventListener('keydown', this.handleButtonKeydown.bind(this));
       this.buttonNode.addEventListener('click', this.handleButtonClick.bind(this));
+      this.buttonNode.addEventListener('focus', this.handleButtonFocus.bind(this));
+      this.buttonNode.addEventListener('blur', this.handleButtonBlur.bind(this));
+      this.buttonNode.addEventListener('pointerenter', this.handleButtonPointerenter.bind(this));
+      this.buttonNode.addEventListener('pointerout', this.handleButtonPointerout.bind(this));
       this.domNode.addEventListener('focusin', this.handleFocusin.bind(this));
       this.domNode.addEventListener('focusout', this.handleFocusout.bind(this));
-      window.addEventListener('mousedown', this.handleBackgroundMousedown.bind(this), true);
+      window.addEventListener('pointerdown', this.handleBackgroundPointerdown.bind(this), true);
+
+    },
+    renderTooltip: function(attachNode, buttonNode) {
+      var id = 'id-skip-to-tooltip';
+      var accesskey = this.getBrowserSpecificAccesskey(this.config.accesskey);
+      var tooltip = this.config.buttonTooltip;
+
+      this.tooltipLeft = buttonNode.getBoundingClientRect().width;
+      this.tooltipTop  = buttonNode.getBoundingClientRect().height;
+
+      this.tooltipNode = document.createElement('div');
+      this.tooltipNode.setAttribute('role', 'tooltip');
+      this.tooltipNode.id = id;
+      this.tooltipNode.classList.add('skip-to-tooltip');
+
+      if (this.isNotEmptyString(accesskey)) {
+        tooltip = this.config.buttonTooltipAccesskey.replace('$key', accesskey);
+      }
+
+      if (this.isEmptyString(tooltip)) {
+        // if there is no tooltip information
+        // do not display tooltip
+        this.config.enableTooltip = false;
+      } else {
+        this.tooltipNode.textContent = tooltip;
+        buttonNode.setAttribute('aria-describedby', id);
+      }
+
+      attachNode.appendChild(this.tooltipNode);
+      this.tooltipNode.style.left = this.tooltipLeft + 'px';
+      this.tooltipNode.style.top = this.tooltipTop + 'px';
+
+      // Need to temporarily show the tooltip to get it's height

```

Comment:
```suggestion
      // Need to temporarily show the tooltip to get its height

```

https://www.grammarly.com/blog/its-vs-its/